opcode-table.texi

```diff
@@ -515,7 +575,7 @@ the stack-effect field.
 @tab Call @code{point} with no arguments.
 @item @verb{|0141|}
 @tab @verb{| *97|}
-@tab @verb{|  save-current-buffer-OBSOLETE|}
```

Comment:
I'm not sure we want to change this? The rest looks good to me!

Reply:
Meant to mention/discuss. The OBSOLETE shouts too much. And it also takes up a lot of space. Also, I don't feel comfortable being too version centric here. 

The opcode's name _is_ save-current-buffer. It has a _property_  of being obsolete.  Changing the name because it has a property doesn't feel right to me. We already mark that that property in the decimal opcode number, but I suppose it could be put here instead. If there's a desire to distinguish "no longer generated" versus no longer acceptable by the interpreter (for some version), I suppose we could use another symbol like "+" instead of "*". 

But overall this table has limited amount of space. The just of this branch is put summary data here and link to a place where we are more expansive where we can describe when was the opcode introduced, when was it no longer generated, and when it was no longer valid for interpretation.

Reply:
I agree with your points, but I don't think using the same name for two different opcodes is a good idea, either. I took the "-OBSOLETE" from one of the .el files, I think, but, again, I do agree it is too shouty.

Feel free to go with whatever solution you think is best, obviously. I personally prefer the "*" in the opcode name, but it's not a strong preference.

The real problem here is there is no "official" list of opcode names; bytecode.c and the byte compiler differ, and neither is complete.

Reply:
Ok. We will disambiguate between the opcodes. And I'll move the prefix to the opcode name field. 

As for the discrepancy between bytecode.c and bytecomp.el, a suggestion is to predominantly go with one, unless there's a reason to go with the other.  I think whatever we do is fine as long as we're transparent about what we are doing (i.e. more verbiage somewhere in the doc),